Formula/dhall.rb

```diff
@@ -0,0 +1,23 @@
+require "language/haskell"
+
+class Dhall < Formula
+  include Language::Haskell::Cabal
+
+  desc "Compiler for the Dhall language"
+  homepage "https://github.com/dhall-lang/dhall-haskell"
+  url "https://hackage.haskell.org/package/dhall-1.18.0/dhall-1.18.0.tar.gz"
```

Comment:
Also, would you be able to update this to the latest version?  It's currently `dhall-1.20.0` (which I uploaded to Hackage just a minute ago 🙂 )

Reply:
Yep, will do